### ISSUE TITLE
Default start sketch

### ISSUE THREAD

**Issue by ssd2** [makerbot/ReplicatorG#32]
This is a combined bug report / feature request...

ReplicatorG seems to randomly decide to remember the last sketch it had open and reloads it at next start.  I can't seem to see a pattern in when it remembers a new one just loaded and when it doesn't and loads the previous one instead.

The feature request would be either a button or menu entry for "Return to this sketch on next load" or a preference somewhere for the name of the default sketch (with the default being BLANK probably).

**Comment by makerbot** [comment]
Bug part:

I'll change the pref to remember the most recently opened file instead of trying to store it on shutdown, which it sounds like it's doing.  This is going to be less of a problem with 0014, which doesn't hang as often.

Feature part:

Would it be sufficient to just reopen the most recently opened file?  This is how most programs work, and I'm not sure if we want to challenge the user's preconceptions.  For what it's worth, there's already a "Recent..." submenu in the File menu.

**Comment by ssd2** [comment]
> Would it be sufficient to just reopen the most recently opened file?

No, not really.  That would actually be irritating.  Usually the last part I opened is the last thing I want, and if I did want it, (as you hinted) I'd open it from the recently opened files menu.

I'd actually rather it came up blank than have the last part.

I actually have a calibration procedure that I'd rather have it open by default, if that was possible.  Ideally, if it is going to remember what it last opened, we'd get a preference that lets us change that, and lets us decide if we want it to remember it or just use the same one every time.
